### ISSUE TITLE
How to instantiate?

### ISSUE THREAD

**Issue by nodesocket** [aurbano/robinhood-node#4]
The README.md shows:

````javascript
rh = Robinhood(credentials);
````

Though seems like it should be:

````javascript
Robinhood(credentials, function(error, broker) {
    if(error) {
        return callback(error);
    }

    callback(null, broker);
});
````

What is recommended?

**Comment by dustinmoorenet** [comment]
Waiting for login worked for me.

```javascript
var Robinhood = require('robinhood');

// Initialize
var trader = Robinhood(
    {
        username: 'user',
        password: 'password'
    },
    function() {
        trader.quote_data('GOOG', function(err, httpResponse, body){
            if (err) {
                console.error(err);
                return;
            }
            console.log('Quote data:', body);
        });
    }
);
```

**Comment by aurbano** [comment]
published v0.1.0 that should fix this
